source/settings/plugin/fts-plugin.rst

```diff
@@ -300,6 +300,47 @@ See also :ref:`fts_tokenization`
 
 .. _plugin-fts-setting-fts_index_timeout:
 
+``fts_header_excludes`` , ``fts_header_includes``
+-----------------------------------------------------
```

Comment:
These need their own labels, i.e. `.. _plugin-fts-setting-fts_header_excludes` and same for includes. Also you need to move all of this stuff before the fts-index-timeout label.

Also .. I think the includes and excludes need to be described separately. Especially once https://github.com/dovecot/documentation/pull/344 gets merged it's a requirement, so might as well do it already. I think the excludes could have the full explanation, while includes has only versionadded + default + values + link to the excludes setting saying it's all explained there.